service/src/test/java/tds/config/web/endpoints/HealthControllerTest.java

```diff
@@ -0,0 +1,29 @@
+package tds.config.web.endpoints;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class HealthControllerTest {
```

Comment:
This doesn't seem related to that common changes.  Was this something you just noticed was missing while you were in there?